features/backend/products.feature

```diff
@@ -108,32 +107,30 @@ Feature: Products
     Scenario: Creating product with string attribute
         Given I am on the product creation page
         When I fill in the following:
-            | Name        | Manchester United tee   |
+            | Name        | FC Barcelona tee        |
```

Comment:
As the original author of this scenario, I vote for reverting this change because of sentimental value.